dependencyManagement/build.gradle.kts

```diff
@@ -16,8 +16,8 @@ val DEPENDENCY_BOMS = listOf(
   "com.squareup.okio:okio-bom:3.7.0", // applies to transitive dependencies of okhttp
   "io.grpc:grpc-bom:1.60.1",
   "io.netty:netty-bom:4.1.104.Final",
+  "io.zipkin.reporter2:zipkin-reporter-bom:3.0.2",
   "io.zipkin.brave:brave-bom:5.17.0",
-  "io.zipkin.reporter2:zipkin-reporter-bom:2.17.1",
```

Comment:
note: the order matters until Brave 6 which doesn't assign a zipkin-reporter dependency version